gatsby-node.js

```diff
@@ -56,7 +62,22 @@ exports.onPostBuild = async function(
     await Promise.all(chunkJobs);
 
     if (settings) {
-      const { taskID } = await indexToUse.setSettings(settings);
+      // Account for forwardToReplicas:
+      const extraModifiers = forwardToReplicas ? { forwardToReplicas } : {};
+
+      const { replicas, ...rest } = settings;
+
+      let adjustedSettings = settings;
+
+      // If we're building replicas, we don't want to add them to temporary indices
+      if (indexToUse === tmpIndex) {
+        adjustedSettings = { ...rest };
+      }
+
+      const { taskID } = await indexToUse.setSettings(
+        adjustedSettings,
+        extraModifiers
+      );
```

Comment:
```suggestion
      // If we're building replicas, we don't want to add them to temporary indices
      const { replicas, ...adjustedSettings } = settings;

      const { taskID } = await indexToUse.setSettings(
        indexToUse === tmpIndex ? adjustedSettings : settings,
        extraModifiers
      );
```

I feel like this is even simpler still, and still the same logic

Reply:
Perfect! Yes, that's much better.

Reply:
@bruce773, seems like I can't commit the suggestion, could you do that so I can merge this PR? Thanks!